brave-lists/https-upgrade-exceptions-list.txt

```diff
@@ -63,6 +63,7 @@ fuw.edu.pl
 fxmconnect.com
 genki365.net
 globalnoticias.pt
+go.microsoft.com
```

Comment:
This one works fine on HTTPS, it's the other one that throws a TLS error:
![Screenshot from 2024-05-02 17-06-03](https://github.com/brave/adblock-lists/assets/167821/231168c3-1a33-4830-af77-9fbbb385821a)